### PR TITLE
code ready

### PR DESCRIPTION
**Pull request by ketilaa**

### CODE REVIEW

File: cristin-funding/build.gradle [hunk 1]
```diff
@@ -0,0 +1,43 @@
+dependencies {
+    implementation project(":cristin-commons")
+    implementation libs.nva.json
+    implementation libs.nva.core
+    implementation libs.nva.language
+//    implementation libs.aws.sdk2.core
```
Comment (torarnet) [inline cristin-funding/build.gradle:6]: Remove commented deps?

File: cristin-funding/src/test/resources/success.json [hunk 2]
```diff
@@ -0,0 +1,3972 @@
+[
+  {
+    "code": "EC/FP7",
```
Comment (torarnet) [inline cristin-funding/src/test/resources/success.json:3]: So this is where all the lines comes from. I thought I had a very big PR but 4/5 of lines is from this file. 

Do you need every entry in cristin? Does it load fast when testing?

File: template.yaml [hunk 3]
```diff
@@ -473,6 +473,36 @@ Resources:
             Path: /person/{id}/employment/{employmentId}/
             Method: patch
 
+  NvaCristinListFundingSourcesFunction:
+    Description: Function for listing all funding sources
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: cristin-funding
+      Handler: no.unit.nva.cristin.funding.sources.query.ListFundingSourcesHandler::handleRequest
+      Role: !GetAtt ReadCristinAuthenticationRole.Arn
```
Comment (torarnet) [inline template.yaml:482]: You dont need this for an open endpoint

File: template.yaml [hunk 4]
```diff
@@ -473,6 +473,36 @@ Resources:
...
+            Path: /funding-sources
+            Method: get
+
+  NvaCristinFetchFundingSourceFunction:
+    Description: Function for fetching a funding source by identifier
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: cristin-funding
+      Handler: no.unit.nva.cristin.funding.sources.fetch.FetchFundingSourceHandler::handleRequest
+      Role: !GetAtt ReadCristinAuthenticationRole.Arn
```
Comment (torarnet) [inline template.yaml:497]: Same

File: template.yaml (hunk 3 above)
Comment (torarnet) [reply template.yaml:482]: It is what allows function to add auth header when doing authorized requests to cristin

**Review by torarnet — DISMISSED** [review]
Approved with minor comments

File: cristin-funding/src/test/resources/success.json (hunk 2 above)
Comment (ketilaa) [reply cristin-funding/src/test/resources/success.json:3]: I don't need them all. I will reduce it.

File: template.yaml (hunk 3 above)
Comment (ketilaa) [reply template.yaml:482]: I will delete.

File: template.yaml (hunk 4 above)
Comment (ketilaa) [reply template.yaml:497]: I will delete

**Pull request merged**